baselines/logging/src/v2/logging_service_v2_client_config.json.baseline

```diff
@@ -0,0 +1,46 @@
+{
+  "interfaces": {
+    "google.logging.v2.LoggingServiceV2": {
+      "retry_codes": {
+        "non_idempotent": [],
+        "idempotent": [
+          "DEADLINE_EXCEEDED",
+          "UNAVAILABLE"
+        ]
+      },
+      "retry_params": {
+        "default": {
+          "initial_retry_delay_millis": 100,
+          "retry_delay_multiplier": 1.3,
+          "max_retry_delay_millis": 60000,
+          "initial_rpc_timeout_millis": 60000,
+          "rpc_timeout_multiplier": 1,
+          "max_rpc_timeout_millis": 60000,
+          "total_timeout_millis": 600000
+        }
+      },
+      "methods": {
+        "DeleteLog": {
+          "retry_codes_name": "non_idempotent",
+          "retry_params_name": "default"
+        },
+        "WriteLogEntries": {
+          "retry_codes_name": "non_idempotent",
+          "retry_params_name": "default"
```

Comment:
Bundling settings are missing here. It won't work without them.
See here: https://github.com/googleapis/nodejs-logging/blob/a3fb0f3b55583220883fb83504f94f57cf907267/src/v2/logging_service_v2_client_config.json#L42-L46

Reply:
done!